local-cli/generator-wpf/templates/src/AppReactPage.cs

```diff
@@ -14,7 +15,7 @@ public override string JavaScriptBundleFile
         {
             get
             {
-                return "ms-appx:///ReactAssets/index.windows.bundle";
+                return AppDomain.CurrentDomain.BaseDirectory + "ReactAssets/index.windows.bundle";
```

Comment:
can this be made to match the inline getter syntax in Playground (or vice versa)?